src/sp_utils.h

```diff
@@ -84,7 +84,7 @@ void unhook_functions(HashTable *ht);
 int hook_regexp(const sp_pcre *, HashTable *, zif_handler);
 bool check_is_in_eval_whitelist(const char* function_name);
 int sp_log_request(zend_string const* const restrict folder, zend_string const* const restrict text_repr);
-#define sp_zend_string_equals(s1, s2) zend_string_equals((zend_string*)s1, (zend_string*)s2)
+#define sp_zend_string_equals(s1, s2) zend_string_equals((const zend_string*)s1, (const zend_string*)s2)
```

Comment:
Unfortunately, on PHP7.0, `zend_string_equals` takes a pair of non-`const` `zend_string`.

Reply:
It's not an compile error, and building against php 7.0 already produces a bunch of warnings, see https://github.com/jvoisin/snuffleupagus/actions/runs/9291505566/job/25570174957?pr=492.

Should the warnings be tweaked for php 7.0?

Reply:
Sigh. Let's get this merged, as it doesn't really make things worse. We can always sprinkle some ifdef around afterwards.